input-output-hk/react-polymorph
# [DDW-658] Custom select value and Stepper component

This PR introduces Custom select value renderer that can replace selected value default input rendered value. Also this PR introduces Stepper component as steps progress bar indicator.

## How to Test

- Custom Select value (`Select -> custom select value template`) - Select any select option and both label and value should appear as selected value
- Go to `Stepper` section and walk through different scenarios.

 **Note:**  Both tests are related to https://zpl.io/aBRGAeA step 1 _"choose wallet"_ step

## Screenshots

_Custom Select Value_

<img width="1184" alt="Custom select value" src="https://user-images.githubusercontent.com/18653950/59414373-ab151180-8dc1-11e9-9e4a-b1b84dd16eb0.png">


_Stepper Component_

<img width="1181" alt="Stepper Component" src="https://user-images.githubusercontent.com/18653950/59414398-b9fbc400-8dc1-11e9-992b-932aaccb0f4f.png">
